navigator/runtime-compose/api/navigator-runtime-compose.api

```diff
@@ -23,7 +23,6 @@ public final class com/freeletics/mad/navigator/compose/NavDestination$Screen :
 
 public final class com/freeletics/mad/navigator/compose/NavHostKt {
 	public static final fun NavHost (Lcom/freeletics/mad/navigator/BaseRoute;Ljava/util/Set;Landroidx/compose/runtime/Composer;I)V
-	public static final fun getLocalNavController ()Landroidx/compose/runtime/ProvidableCompositionLocal;
```

Comment:
This and the other changes to `.api` files below are because https://github.com/Kotlin/binary-compatibility-validator/releases/tag/0.10.0 is now properly excluding getters from annotated properties